From: Hilko Bengen <bengen@debian.org>
Date: Sat, 5 Jan 2013 12:58:30 +0100
Subject: Fix packagelist

- ntfsprogs has been merged with ntfs-3g -- and it has been an empty
  transitional package for some time now.
- libyajl is needed for ldm
- libcap2 is needed for guestfs
---
 appliance/packagelist.in |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/appliance/packagelist.in b/appliance/packagelist.in
index 3ad343b..6fd9438 100644
--- a/appliance/packagelist.in
+++ b/appliance/packagelist.in
@@ -62,14 +62,15 @@
   hfsplus
   iproute
   libaugeas0
+  libcap2
   libhivex0
   libyajl2
   linux-image
   nilfs-tools
   ntfs-3g
-  ntfsprogs
   openssh-client
   reiserfsprogs
+  sysvinit /* for /sbin/reboot */
   ufsutils
   vim-tiny
   xz-utils
